examples/plot_pp_study.py

```diff
@@ -0,0 +1,362 @@
+
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+================================
+Predictive Policing Case Study
+================================
+"""
+# %%
+#
+#
+# Contents
+# ========
+#
+# 1. Overview
+# 2. Background
+# 3. Fairlearn Results
+# 4. Problem area & Sociotechnical context
+# 5. Conclusion
+#
+# Overview
+# --------
+#
+# In this document we outline a sample sociotechnical case study - using a
+# hypothetical predictive policing scenario - based on data found in the
+# `Boston housing dataset <https://www.kaggle.com/c/boston-housing>`_
+# which we use in our scenario to predict per capita crime rate.
+# Discussion around the known fairness issues associated with this dataset
+# is included `here
+# <https://medium.com/@docintangible/racist-data-destruction-113e3eff54a8>`_.
+# For further questions about this document please contact Bruke Kifle
+# (@brkifle) or Michael Amoako (@miamoako). The full notebook containing
+# the code below is available upon request.
+#
+# Background
+# ----------
+#
+# You are a contractor working with the Boston Police Department and
+# advising them in their effort to add predictive analytics and
+# intelligence to their existing policing practices. One idea the Boston
+# Police Department has is to adopt an algorithmic decision-making tool
+# that would allow them to determine how heavily different regions in
+# Boston should be policed, based on the model’s predicted crime rate
+# classification. Such practice is known as ‘predictive policing’ – the
+# use of analytical techniques in law enforcement to predict ‘potential’
+# criminal activity and take proactive measures to pre-empt crime. In this
+# example, assume that if the model’s output is above the defined
+# threshold, the described area is more heavily policed. Some key
+# characteristics of the scenario:
+#
+#
+# * Stakeholders: police officers, members of the community (especially
+#   people who may be more affected by predictive policing than others),
+#   contractor / data scientist
+#
+# * Harms: overpolicing of neighborhoods can
+#   lead to disproportionate effect on communities in these neighborhoods
+#   (perhaps exacerbated by feedback loop)
+#
+# * Notes: Feedback loops!
+#   Prediction on behavior based on circumstances; perhaps useful for
+#   aggregate observations about behavior but less so for individual
+#   predictions
+#
+# * Dangerous to assume that ‘crime’ is a single phenomenon.
+#   There are different kinds of crime, so also need to make sure that data
+#   match the crime being predicted.
+#
+# For this case study, we are utilizing the Boston housing value dataset
+# (described here). Originally, this dataset includes the following variables:
+#
+# .. list-table::
+#    :header-rows: 1
+#    :widths: 5 20
+#    :stub-columns: 1
+#
+#    *  - Variable
+#       - Description
```

Comment:
For most of these variables, it's not clear why these might reasonably be used to predict crime